edx-platform/pearson-kaust-theme/lms/templates/learner_profile/learner-achievements-fragment.html

```diff
@@ -0,0 +1,89 @@
+## mako
+
+<%page expression_filter="h"/>
+
+<%namespace name='static' file='/static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<div class="learner-achievements">
+    % if course_certificates or own_profile:
+        <h3 class="u-field-title">${_("Course Certificates")}</h3>
+        % if course_certificates:
+            % for certificate in course_certificates:
+                <%
+                certificate_url = certificate['download_url']
+                course = certificate['course']
+
```

Comment:
@Squirrel18, can we remove this blank line?

Reply:
@ivanvgh I'll leave it as the original template: https://github.com/Pearson-Advance/edx-platform/blob/pearson-release/juniper.master/openedx/features/learner_profile/templates/learner_profile/learner-achievements-fragment.html